vault/external_tests/identity/login_mfa_okta_test.go

```diff
@@ -6,20 +6,56 @@ package identity
 import (
 	"context"
 	"fmt"
+	"os"
 	"reflect"
+	"strings"
 	"testing"
+	"time"
 
 	"github.com/hashicorp/vault/api"
+	"github.com/hashicorp/vault/helper/constants"
+	"github.com/hashicorp/vault/helper/testhelpers"
+	logicaltest "github.com/hashicorp/vault/helper/testhelpers/logical"
 	"github.com/hashicorp/vault/helper/testhelpers/minimal"
 )
 
-var (
-	org_name  = "<okta org name>"
-	api_token = "<okta api token>"
-)
+// To run these tests, set the following env variables:
+// VAULT_ACC=1
+// OKTA_ORG=dev-219337
+// OKTA_API_TOKEN=<generate via web UI, see Confluence for login details>
+// OKTA_USERNAME=test3@example.com
+// OKTA_PASSWORD=<find in 1password>
+//
+// You will need to install the Okta client app on your mobile device and
+// setup MFA in order to use the Okta web UI. This test does not exercise
+// MFA however (which is an enterprise feature), and therefore the test
+// user in OKTA_USERNAME should not be configured with it. Currently,
+// test3@example.com is not a member of testgroup, which is the group with
+// the profile that requires MFA. If you need to use a different group name
+// for the test group, you can set:
+// OKTA_TEST_GROUP=alttestgroup
```

Comment:
I'm not sure I understand this part of the comment. I don't see any policy onfiguration for mfa in the test directly below this one (the TestOktaEngineMFA one which is the one that seems to care about this new group ENV var).

I didn't see where this was used in either of the other two tests. Am I missing something you saw here? I didn't need to set this to get a PASS from the test (although as described above the MFA behavior was confusing for other reasons).

Reply:
`OKTA_TEST_GROUP` is only used in this test, not the other 2 tests. In Okta, I configured my vault App's Authentication Policy to either require MFA when running the tests. You can also do that with the Global Session Policy.

I think the point of the tests in this file are:
- `TestOktaEngineMFA`: Test Okta Push Verify MFA support via Okta Auth Method (not via Login MFA). The Okta Auth method only fully supports MFA with Push Verify, not TOTP.
- `TestInteg_PolicyMFAOkta`: Test Okta MFA support via Policy MFA (not via Okta Auth Method or Login MFA). This only supported Push Verify before, and I added TOTP to the test, assuming it somehow reused the Login MFA stuff.
- `TestInteg_LoginMFAOkta`: Test Okta MFA support via Login MFA (not via Okta Auth Method or Policy MFA). This is the primary test for this PR.

Honestly, the last test is the only test that "belongs" in the `login_mfa_okta_test.go` file. Maybe this test should move to `builtin/credential/okta`? Or maybe they are all here to make you think about the other Okta MFA support in Vault when you make changes?

I guess my confusion about why these tests are grouped in this file is part of why the comments I put here also confused you.